modules/installation-configure-proxy.adoc

```diff
@@ -62,6 +63,9 @@ ifeval::["{context}" == "installing-aws-china-region"]
 :aws:
 :aws-china:
 endif::[]
+ifeval::["{context}" == "installing-alibaba-vpc"]
```

Comment:
You have added an `ifeval` for your VPC assembly, but I do not see any specific content that would require the condition.

Reply:
The file has been removed from this PR (as expected).